integration_tests/testsuite/deploy_app.py

```diff
@@ -21,6 +21,8 @@
 import subprocess
 import sys
 
+import test_util
+
 
 def _cleanup(appdir):
```

Comment:
Yeah I should have put a comment in here saying I was working on it when I made this PR. Coming soon